examples/L5PC/optimisation.sbatch

```diff
@@ -33,6 +33,7 @@ echo "${OPT_EMODEL},${OPT_SEED},${GITHASH},${SLURM_JOB_ID},${SLURM_NTASKS},${RUN
 export IPYTHON_PROFILE=optimisation__${SLURM_JOB_ID}_$(hostname)
 export USEIPYP=1
 export IPYTHONDIR="`pwd`/.ipython"
+export NEURON_MODULE_OPTIONS="-nogui"
```

Comment:
This might be needed in analysis.sbatch, export.sbatch as well. What do you think? NEURON is launched during these steps.

Reply:
Good point, I added it in those scripts as well.